src/plugins/accounting.cpp

```diff
@@ -140,3 +140,37 @@ void split_string_and_push_back (const char *list,
         vec.push_back (substr);
     }
 }
+
+
+int get_queue_info (char *queue,
+                    std::vector<std::string> permissible_queues,
+                    std::map<std::string, struct queue_info> queues)
```

Comment:
Consider taking the last two arguments as const references? Because I think at the moment they're passed as copies, which is a bit wasteful.

Reply:
Agreed! Thanks for pointing this out. Just pushed up a fix to change those to const references.